README.md

```diff
@@ -1 +1,78 @@
+
+## Install
+
+```
+npm install testcafe-angular-selectors
+```
+
+## Angular1 Selector extensions
+
+### byBinding
+Find elements by text binding. Does a partial match, so any elements bound to variables containing the input string will be returned.
+```js
+Angular1Selector.byBinding(expression, parentSelector)
+```
+Parameter                   | Description
+--------------------------- | -----------
+expression                        |  The JavaScript expression to which the element's `textContent` is bound.
+parentSelector&#160;*(optional)*  | A TestCafe [selector](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html). If specified, TestCafe will search for the target element among the descendants of the element identified by this selector.
+
+NOTE: we don't support a deprecated syntax `Angular1Selector.byBinding('{{person.name}}')`
+
+### byExactBinding
+Find elements by exact binding.
+```js
+Angular1Selector.byExactBinding(expression, parentSelector)
+```
+Parameter                   | Description
+--------------------------- | -----------
+expression                        |  The JavaScript expression to which the element's `textContent` is bound.
+parentSelector&#160;*(optional)*  | A TestCafe [selector](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html). If specified, TestCafe will search for the target element among the descendants of the element identified by this selector.
+
+### byModel
+Find elements by 'ng-model' expression
+```js
+Angular1Selector.byModel(expression, parentSelector)
+```
+Parameter                   | Description
+--------------------------- | -----------
+expression                        | The JavaScript expression used to bind a property on the scope to an input, select, textarea (or a custom form control).
+parentSelector&#160;*(optional)*  | A TestCafe [selector](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html). If specified, TestCafe will search for the target element among the descendants of the element identified by this selector.
+
+### byOptions
+                 
+Find elements by 'ng-options' expression.
+```js
+Angular1Selector.byOptions(expression, parentSelector)
+```
+Parameter                   | Description
+--------------------------- | -----------
+expression                        | The JavaScript expression used to generate a list of <option> elements for the <select> element.
+parentSelector&#160;*(optional)*  | A TestCafe [selector](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html). If specified, TestCafe will search for the target element among the descendants of the element identified by this selector.
+
+### byRepeater
+Find elements by repeater. Does a partial match, so any elements bound to variables containing the input string will be returned.
```

Comment:
> any elements bound to variables containing the input string will be returned

Is this true for a repeater as well? Or we should probably write

`so any element that matches the specified template will be returned.`